src/hotspot/share/memory/arena.hpp

```diff
@@ -52,11 +52,12 @@ class Chunk: CHeapObj<mtChunk> {
   enum {
     // default sizes; make them slightly smaller than 2**k to guard against
     // buddy-system style malloc implementations
+    // Note: please keep these constants 64-bit aligned.
```

Comment:
Can you add a static_assert(is_aligned(slack, ARENA_ALIGNMENT)?

Reply:
Sure!